lib/plugins/aws/deploy/lib/checkForChanges.js

```diff
@@ -98,7 +98,12 @@ module.exports = {
     });
 
     return BbPromise.all(getFunctionResults)
-      .then(_.min)
+      .then(results =>
+        results.reduce((currentMin, date) => {
+          if (!currentMin || date < currentMin) return date;
+          return currentMin;
+        }, undefined)
```

Comment:
We can improve it further, let;s replace `undefined` with `null` and remove `then(..)`  that follows it :)

Reply:
Thanks a lot! I've learned so much from these PR reviews. :D

Reply:
Great to hear that :)